src/doc/rustdoc/src/lints.md

```diff
@@ -285,6 +285,58 @@ warning: unclosed HTML tag `h1`
 
 warning: 2 warnings emitted
 ```
+                
+## invalid_rust_codeblock
+
+This lint **warns by default**. It detects Rust code blocks in documentation
+examples that are invalid (e.g.: empty, not parsable as Rust). For example:
+
+```rust
+/// Empty code block, with and without Rust:
+///
+/// ```
+/// ```
+///
+/// ```rust
+/// ```
+///
```

Comment:
I think this example might be a little much - it's pretty clear what the error would be and I think it's better to make the docs concise. I could be convinced otherwise though.

Reply:
This file being the reference for rustdoc-lints, I would prefer it to be as complete as possible, but that's not something critical IMO, just a nice thing

Reply:
It's the reference for which lints *exist*, but not for all warnings emitted by the lints. For example intra-doc links only have a couple of the warnings emitted even though there are many more forms of the error: https://doc.rust-lang.org/nightly/rustdoc/lints.html#broken_intra_doc_links

I don't feel super strongly though, up to you if you want to change it.

Reply:
I left one instance of an empty code block and one for an invalid code block